packages/dev/docs/pages/blog/creating-a-pointer-friendly-submenu-experience.mdx

```diff
@@ -0,0 +1,108 @@
+
+<SubmenuAtan2FromPointer
+  role="img"
+  aria-label="Diagram showing triangles formed between pointer and various points on submenu" />
+
+If the pointer’s delta angle is **between** the top and bottom angles, we know the user is moving their pointer in the direction of the submenu.
+
+<p style={{textAlign: 'center', fontSize: 'xx-large'}}>Θ<sub>top</sub> > Θ<sub>pointer</sub> > Θ<sub>bottom</sub></p>
+
+## Time
+
+We also want to consider the pointer’s movement over time when predicting the user’s intent. A user typically increases the **speed** of their pointer when moving towards the submenu, then decreases it as they reach their target. The user might change their mind on the way, however, and slow down to a stop or keep browsing other options in the parent menu. If their pointer speed dips below a certain threshold before arriving at the submenu, we could assume they're no longer intending to go to the submenu.
+We could check the pointer’s speed throughout its journey with our knowledge of what speeds to expect at the different stages along the way. We can even take it a step further by using our knowledge of the [Accot–Zhai steering law](https://en.wikipedia.org/wiki/Steering_law) to predict the time it may take the user to reach the target based on the width of the “tunnel” to the submenu.
+Alternatively, we can go with a simpler solution and implement a **timeout**. If the pointer hasn’t moved after a certain amount of time, we can assume they are no longer intending to go to the submenu. Since users with motor impairments may take more time to move their pointer, we should lean towards using a larger timeout value. Although our timeout solution is more simple than incorporating the pointer’s speed, we found that it still results in a good user experience.
+
+## Fault Tolerance
+
+Since our users aren’t perfect, we want to build in some fault tolerance in case their actions are outside the scope we have defined so far, but not by enough to invalidate their intent. Here are some ways we can do that:
+
+* Widen our range of allowed angles: Add radians to our top and bottom angles.
+* Allow N invalid movements: Require that the user make two (or some larger constant) consecutive invalid pointer movements before closing the submenu. Users who experience [tremors](https://en.wikipedia.org/wiki/Tremor) may involuntarily move their pointer in other directions, so it is important to include this feature.
+
+## Optimizations
+
+We can introduce a few performance optimizations:
+
+* Throttle: Most devices refresh their screens 60 times per second. This means that we don't need to do these measurements more frequently than every 16 ms (1 second / 60 = 16.66 ms). We can keep timestamps and skip checks if enough time hasn't passed. We can also experiment with doing checks even less frequently while still maintaining a good user experience. Lowering the sample rate would also provide more accurate predictions for users who experience tremors.
```

Comment:
I think you could make "throttle" and "avoid unnecessary work" bold

and same goes for the "Alternatives" section